rcl/test/rcl/test_graph.cpp

```diff
@@ -101,6 +104,12 @@ class CLASSNAME (TestGraphFixture, RMW_IMPLEMENTATION) : public ::testing::Test
     ret = rcl_wait_set_init(
       this->wait_set_ptr, 0, 1, 0, 0, 0, 0, this->context_ptr, rcl_get_default_allocator());
     ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+
+    rmw_topic_info_array_t valid_topic_info_array = {
+      0,     /*count*/
+      nullptr     /*info_array*/
+    };
+    this->topic_info_array = &valid_topic_info_array;
```

Comment:
you can't do this.
`valid_topic_info_array` is stack allocated and you're leaking a pointer of it to somewhere else.

ideally you'd do 
```c++
this->topic_info_array = std::make_unique<rmw_topic_info_array_t>();
```
However this requires `topic_info_array` to be a `unique_ptr`.
If you **must** use a raw pointer here, use `new`, however the class should probably be changed to use a `unique_ptr` some time in the future.